.github/actions/get-issue-url-and-version-number-v1/src/run.ts

```diff
@@ -0,0 +1,44 @@
+import type { Core, GitHub, Issue } from './types'
+import { getExecOutput } from '@actions/exec'
+
+export default async function run(core: Core, github: GitHub) {
+  const { repo, owner } = github.context.repo
+  try {
+    const {
+      stdout: rawIssue,
+      stderr: issueError,
+      exitCode: issueExitCode
+    } = await getExecOutput(
+      `gh issue list --repo ${owner}/${repo} --label release --state open --json url,title`
+    )
+
+    if (issueExitCode) {
+      throw new Error(`Error getting issue:\n${issueError}`)
+    }
+
+    const [issue] = JSON.parse(rawIssue.trim()) as Issue[]
```

Comment:
This assumes that the `gh issue list` command will always find an issue that matches. We should build in guards just in case it doesn't